proxy/nginx.conf.template

```diff
@@ -228,11 +228,21 @@ server {
   location /api/application {
     include     proxy_params;
     proxy_pass  $enduserUrl;
+
+    location /api/application/attachments {
+      client_max_body_size 0;
```

Comment:
:grimacing: I'd much rather prefer _some_ reasonable limit instead of unlimited. Most optimally, I'd have it as a configurable parameter but can cope with something reasonable.

Reply:
I guess realistically evaka will never accept files over 100MB, but I just don't want this to be configured twice. How about the move the 10MB limit to proxy and remove it from service?

Reply:
I suppose we inevitably have to have some implementation details on the proxy level (i.e. the path `/api/application/attachments`), so I can live with that.

The only gripe I have is that IMO the service should still block unreasonably sized uploads even if it wasn't behind this particular proxy (and architecture-wise it shouldn't even know about its existence). Personally I'd just do the double configuration with a high limit at the proxy level and a "use case" limit at the service level but as said, I can live with it just being a proxy configuration. Practically the chance of us replacing the proxy with something else is rather low. Though, the other municipalities might want to use their own :thinking:.

Reply:
Alright, I'll just a large limit in the proxy and keep the service limit intact.